styles/accessibility.css

```diff
@@ -1,5 +1,5 @@
-.nav-link:focus-visible, :focus-visible {
+.nav-link:focus-visible, :focus-visible, .btn:focus-visible {
```

Comment:
Instead of this override approach, can we add this into `buttons.css` and use the bootstrap provided variable (e.g. `--bs-btn-focus-box-shadow`)?

Reply:
moved style to button.css and defined color for BS variable